plugins/package-managers/sbt/src/main/kotlin/Sbt.kt

```diff
@@ -70,7 +75,22 @@ class Sbt(
 
         logger.info { "Determined '$workingDir' as the $managerName project root directory." }
 
-        checkConfiguredSbtVersions(workingDir)
+        val sbtVersions = getBuildSbtVersions(workingDir)
+        when {
+            sbtVersions.isEmpty() ->
+                logger.info { "The build does not configure any $managerName version to be used." }
+
+            sbtVersions.size == 1 ->
+                logger.info { "The build configures $managerName version ${sbtVersions.first()} to be used." }
+
+            else ->
+                logger.warn { "The build configures multiple different $managerName versions to be used: $sbtVersions" }
+        }
+
+        val lowestSbtVersion = sbtVersions.firstOrNull()
+        require(lowestSbtVersion?.isLowerThan(Semver(LOWEST_SUPPORTED_SBT_VERSION)) == false) {
```

Comment:
The condition will evaluate to `false` if `lowestSbtVersion == null`. Is this intended or should a different error message be printed in this case?

Reply:
Thanks, fixed the condition to be `!= true`.